gear/user_management/src/python/user_app/run.py

```diff
@@ -31,22 +31,28 @@
 class UserManagementVisitor(GearExecutionEnvironment):
     """Defines the user management gear."""
 
-    def __init__(self,
-                 admin_id: str,
-                 client: ClientWrapper,
-                 user_filepath: str,
-                 auth_filepath: str,
-                 email_source: str,
-                 comanage_config: Configuration,
-                 comanage_coid: int,
-                 force_notifications: bool = False):
+    def __init__(
+        self,
+        admin_id: str,
+        client: ClientWrapper,
+        user_filepath: str,
+        auth_filepath: str,
+        email_source: str,
+        comanage_config: Configuration,
+        comanage_coid: int,
+        parameter_store: ParameterStore,
+        redcap_path: str,
+        force_notifications: bool = False,
+    ):
         super().__init__(client=client)
         self.__admin_id = admin_id
         self.__user_filepath = user_filepath
         self.__auth_filepath = auth_filepath
         self.__email_source = email_source
         self.__comanage_config = comanage_config
         self.__comanage_coid = comanage_coid
+        self.__parameter_store = parameter_store
```

Comment:
replace parameter_store and redcap_path with the full dictionary of project parameters